Libs/Visualization/VTK/Widgets/ctkVTKAbstractView.h

```diff
@@ -51,6 +51,7 @@ class CTK_VISUALIZATION_VTK_WIDGETS_EXPORT ctkVTKAbstractView : public QWidget
   Q_PROPERTY(QColor backgroundColor2 READ backgroundColor2 WRITE setBackgroundColor)
   Q_PROPERTY(bool gradientBackground READ gradientBackground WRITE setGradientBackground)
   Q_PROPERTY(bool renderEnabled READ renderEnabled WRITE setRenderEnabled)
+  Q_PROPERTY(bool pauseRender READ isRenderPaused WRITE pauseRender)
```

Comment:
Have you tried this from Python? I'm afraid that we cannot do `wasPaused = view.pauseRender(True)` because properties can be set like this: `view.pauseRender = True`. If this is the case then it may be better to specify isRenderPaused and pauseRender as Q_INVOKABLE and remove this Q_PROPERTY.